src/ffi.rs

```diff
@@ -59,7 +59,8 @@ pub struct BlindedMessage {
 pub extern "C" fn blind(
     message: *const Buffer,
     seed: *const Buffer,
-    blinded_message: *mut BlindedMessage,
+    blinded_message_out: *mut Buffer,
+    blinding_factor_out: *mut *mut Token<PrivateKey>,
```

Comment:
The documentation should also be updated:
```
/// * blinded_message: Pointer to the memory where the blinded message will be written to
```

and

```
/// The `BlindedMessage.blinding_factor` should be saved for unblinding any
/// signatures on `BlindedMessage.message`
```

IMO also add a doc that says "you should free vec and destroy_token once you're done"

Reply:
Yes to both, will fix!